packages/mapviewer/src/modules/map/components/cesium/CesiumInteractions.vue

```diff
@@ -31,14 +44,32 @@ const visiblePrimitiveLayers = computed(() =>
 onMounted(() => {
     const viewer = getViewer()
     if (viewer) {
+        initialize3dHighlights()
+        viewer.scene.postProcessStages.add(
+            PostProcessStageLibrary.createSilhouetteStage([
+                hoveredHighlightPostProcessor.value,
+                clickedHighlightPostProcessor.value,
+            ])
+        )
         viewer.screenSpaceEventHandler.setInputAction(onClick, ScreenSpaceEventType.LEFT_CLICK)
         viewer.screenSpaceEventHandler.setInputAction(
             onContextMenu,
             ScreenSpaceEventType.RIGHT_CLICK
         )
+        viewer.screenSpaceEventHandler.setInputAction(onMouseMove, ScreenSpaceEventType.MOUSE_MOVE)
     }
 })
+function initialize3dHighlights() {
+    hoveredHighlightPostProcessor.value = PostProcessStageLibrary.createEdgeDetectionStage()
+    hoveredHighlightPostProcessor.value.uniforms.color = Color.BLUE
```

Comment:
Please use our colors defined in `src/modules/map/components/cesium/utils/highlightUtils.js`